src/grimp/adaptors/_layers.py

```diff
@@ -175,193 +174,3 @@ def _module_from_layer(layer: str, container: str | None = None) -> _Module:
     else:
         name = layer
     return _Module(name)
-
-
-def _search_for_package_dependency(
```

Comment:
Removing this dead code is of course not really related to the topic of this PR, just cleaning up. I assume there is no reason that we should be keeping this code anymore?